### ISSUE TITLE
Bot occasionally stops working

### ISSUE THREAD

**Issue by addrum** [ahmed-taj/dep#33]
We're running the bot across our org (_not_ self hosted) and recently it's stopped working and we're not sure why. We've got fairly lengthy PR descriptions but adding "depends on #xx" at the start or end doesn't make a difference, nor does capitalisation or the other key words. We've tried a couple of reinstalls, the first time this seemed to fix things, the second time it hasn't. Please let me know how we can get more information to debug this if required!

We have thought about self-hosting it, but wanted to report here first to see if it was a bug, in which case a self hosted instance might not make a difference.

**Comment by addrum** [comment]
I should add by "not working", I mean we don't get the "dependent" label added nor see the bot in the status checks.

**Comment by z0al** [comment]
Thank you for reporting this. I checked the deployment and it seems down due to:

> This deployment is no longer available.
ZEIT Now 1.0 has been shut down. As part of this, all ZEIT Now 1.0 Deployments are now inaccessible.

I will need to migrate the deployment to Now 2.0. I will try to do it as soon as possible or on the next weekend maybe.

**Comment by leotrs** [comment]
Any updates here? Eager to start using this bot soon!

**Comment by z0al** [comment]
Sorry for the delay. I started working on this and will publish a new version soon-ish. It has been a while since this app received any love so I'm upgrading some dependencies.

**Comment by leotrs** [comment]
Thanks for the update! :)

**Comment by z0al** [comment]
I just re-deployed the App again, but this time using Glitch instead of Vercel. Everything should be working but if you found a problem please open a new issue.
